packages/client/src/generation/TSClient/utils/invalidImportWarning.ts

```diff
@@ -0,0 +1,5 @@
+export const invalidImportWarning = [
+  'Add your client as a `db` dependency to `package.json` and import it as a dependency instead',
```

Comment:
> add ... as a ... dependency and import as a dependency instead
Is this correct?

Reply:
What seems incorrect to you?

Reply:
using "dependency" twice, once for the fact that it is a `dependency` in the `pkg.sjon`, and then how it is imported - also referred to as "dependency".

Reply:
Could you help me reformulate/refine this?

Reply:
I updated the message so it is more packed with all necessary information and steps.